grpclb/src/main/java/io/grpc/grpclb/GrpclbState.java

```diff
@@ -453,9 +460,18 @@ private void useRoundRobinLists(
           eagList.add(new EquivalentAddressGroup(origEag.getAddresses(), eagAttrs));
         }
         if (subchannels.isEmpty()) {
-          // TODO(zhangkun83): remove the deprecation suppression on this method once migrated to
-          // the new createSubchannel().
-          subchannel = helper.createSubchannel(eagList, createSubchannelAttrs());
+          subchannel =
+              helper.createSubchannel(
+                  CreateSubchannelArgs.newBuilder()
+                      .setAddresses(eagList)
+                      .setAttributes(createSubchannelAttrs())
+                      .build());
+          subchannel.start(new SubchannelStateListener() {
+            @Override
+            public void onSubchannelState(ConnectivityStateInfo newState) {
+              // No-op
```

Comment:
I think still need to call `GrpclbState.handleSubchannelState()` otherwise it's never called for pick_first. And the test calling `GrpclbState.handleSubchannelState()` directly for pick_first seems cheating. It might require capturing/storing the listeners and delivering subchannel state with the listeners, could be messy.

Reply:
okay. i use that shortcut because it requires non trivial changes. converted mocked classes to fake/real classes, PTAL.